.github/CONTRIBUTING.md

```diff
@@ -93,9 +93,9 @@ Things you **CAN'T** do:
 As mentioned before, you are expected to follow these specifications in order to make everyone's lives easier. It'll save both your time and ours, by making sure you don't have to make any changes and we don't have to ask you to. Thank you for reading this section!
 
 ### Object Oriented Code
-As BYOND's Dream Maker (henceforth "DM") is an object-oriented language, code must be object-oriented when possible in order to be more flexible when adding content to it. If you don't know what "object-oriented" means, we highly recommend you do some light research to grasp the basics.
+As BYOND's Dream Maker (henceforth "DM") is an object-oriented language, code should be object-oriented when possible in order to be more flexible when adding content to it. If you don't know what "object-oriented" means, we highly recommend you do some light research to grasp the basics.
 
-### All BYOND paths must contain the full path
+### All BYOND paths should contain the full path
```

Comment:
Relative pathing is the work of the devil and should never be allowed under any circumstances

Reply:
its also linted against